app/presenters/hyrax/admin/workflow_role_presenter.rb

```diff
@@ -2,7 +2,7 @@ module Hyrax
   module Admin
     class WorkflowRolePresenter
       def users
-        ::User.all
+        ::User.where(guest: false)
```

Comment:
Now that we've got this in a couple places, it might be good to extract this into the User model as a new class method called `.guests`, but I don't think you need to do that in this PR.